book/website/collaboration/inclusive-events/inclusive-events-comms.md

```diff
@@ -0,0 +1,487 @@
+
+Ask the speaker if they require any support or accommodations and ensure that these are met to their satisfaction so that they are able to deliver their session effectively. 
+For example, ensuring that there is an appropriately trained and qualified sign language interpreter who is able to work with a deaf or hard of hearing speaker. 
+
+Additionally, if your event is supporting a speaker or facilitator with lived experience, consider asking how they wish to be identified in relation to their expertise and experience. 
+For example, many autistic people, and the autistic community as a collective, have expressed a preference for using "Identify-first" language which places the disability first in the description. 
+This is contrasted with "Person-first" language which emphasizes the person before the disability, for example, __Person with epilepsy_. 
+Many people feel strongly about person or identity first language, and it is important to ask that individual about their preference as part of planning their introduction. 
+
+You may notice that we have used both person-first and identity-first language has been used in this chapter. 
+How a person chooses to self-identify is entirely a personal decision, and it is the role of organizers and facilitators to support the choices of the speakers and attendees, 
+as long as it does not contravene the event's Code of Conduct. 
+Consider instead including a statement on the inclusive and compassionate use of both person-first and identity-first language when opening your event. 
+
+(cl-inclusive-events-comms-speaker-preparation)=
+### Speaker Preparation
+Depending on the type of event you are planning, you may want to consider providing guidance for speakers, moderators, and presenters on inclusive communication, 
+and requesting that any content delivered in the session can be sent to attendees before the event. 
+
+If your event has online or streamed sessions, consider developing an information pack and working with your speakers and event facilitators to 
+ensure that content is accessible and inclusive of all attendees. 
+This information pack can contain guidance on how to present so that attendees with sensory processing differences or hearing impairments are able to engage in the session content. 
+For example, recommending that speakers remain as still as possible while they are speaking and presenting, and requesting that online presenters are well-lit with a blurred background 
+will help support attendees who rely on lip and face reading to engage in a session.
+
+You can also include information about the audience and provide guidance on how to effectively communicate to an audience of people with lived experience. 
+Many early career researchers may not have recieved formal training in scientific communication and working with the public, so it can be helpful to emphasize the importance of using plain language and speaking clearly. 
```

Comment:
```suggestion
Many early career researchers may not have received formal training in scientific communication and working with the public, so it can be helpful to emphasize the importance of using plain language and speaking clearly. 
```